examples/point10.cpp

```diff
@@ -116,22 +116,22 @@ int main() {
 	for (int i = 0 ; i < N ; i ++) {
 		// When we decompress data we need to provide where to decode stuff to
 		//
-		las::point10 p;
-		decompressor.decompressWith(decoder, (char *)&p);
+		las::point10 p2;
```

Comment:
Because there's another variable `p` in scope above this loop and MSVC was complaining about it masking.